src/server/views/login.njk

```diff
@@ -37,7 +37,7 @@
               <h3 class="govuk-notification-banner__heading">
                 Email sent
               </h3>
-              <p class="govuk-body">We have sent you an email with the login instructions</p>
+              <p class="govuk-body">We have sent you an email with the login instructions. <br/> If you do not receive an email, contact (someone).</p>
```

Comment:
could you chase on copy for this please? It should be an email address or a role like "the administrator"